uiowa/uiowa
# ListBlock Updates

Relates to https://github.com/uiowa/uiowa/issues/4158

Created draft PR from Makur's branch

## To Test

- Interact with existing views blocks. Check for visual regressions from that of PROD (e.g. filters being revealed that weren't previously revealed).
- Create new people list blocks in v3 and make sure the settings provided are what are rendered.
